composer.json

```diff
@@ -38,9 +38,9 @@
         "oro/doctrine-extensions": "^1.0",
         "ongr/elasticsearch-bundle": "^5.0",
         "sulu/article-bundle": "^1.0@RC",
-        "sulu/automation-bundle": "^1.2",
-        "sulu/redirect-bundle": "^0.1",
-        "sulu/sulu-form-bundle": "^0.3"
+        "sulu/automation-bundle": "dev-develop#b6f6ea6 as 1.3.0",
+        "sulu/redirect-bundle": "dev-master#f072f52 as 0.2.0",
+        "sulu/sulu-form-bundle": "dev-develop#0123f37 as 0.4.0"
```

Comment:
I think I would not fix that to a commit... The bundles should always work, and if we update them, we always have to touch these files 😕